### PR TITLE
MODORDSTOR-61 Receiving history: updates to support check-in flow

### PR DESCRIPTION
**Pull request by pautau-s**
## Purpose
[MODORDSTOR-61](https://issues.folio.org/browse/MODORDSTOR-61) update receiving history to support check-in flow

## Approach
- update acq-models
- receiving history view creation script updated
- Added check for new properties in ReceivingHistoryTest

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/resources/templates/db_scripts/schema.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+      "snippet": "CREATE OR REPLACE VIEW receiving_history_view AS SELECT pieces.id AS id, json_build_object('id', pieces.jsonb->>'id', 'caption', pieces.jsonb->>'caption', 'comment', pieces.jsonb->>'comment','dateOrdered', purchase_order.jsonb->>'dateOrdered', 'checkin', po_line.jsonb->>'checkin_items', 'instanceId', po_line.jsonb->>'instance_id','itemId', pieces.jsonb->>'itemId', 'locationId', pieces.jsonb->>'locationId', 'poLineOrderFormat', po_line.jsonb->>'order_format', 'poLineId', pieces.jsonb->>'poLineId', 'poLineNumber', po_line.jsonb->>'po_line_number', 'purchaseOrderId', po_line.jsonb->>'purchase_order_id', 'poLineReceiptStatus', po_line.jsonb->>'receipt_status', 'receivedDate', pieces.jsonb->>'receivedDate', 'title', po_line.jsonb->>'title', 'receivingNote', details.jsonb->>'receiving_note', 'receivingStatus', pieces.jsonb->>'receivingStatus', 'supplement', pieces.jsonb->>'supplement') AS jsonb FROM pieces LEFT OUTER JOIN po_line ON pieces.jsonb->>'poLineId' = po_line.jsonb->>'id' LEFT OUTER JOIN details ON details.jsonb->>'po_line_id' = po_line.jsonb->>'id' LEFT OUTER JOIN purchase_order ON po_line.jsonb->>'purchase_order_id' = purchase_order.jsonb->>'id';",
```
Comment (craigmcnally) [inline src/main/resources/templates/db_scripts/schema.json]: This has been updated as part of https://github.com/folio-org/mod-orders-storage/pull/90... You'll have to update this once it's been merged...  "Details" has been pulled into poLine.

File: src/test/resources/po_line.sample [hunk 2]
```diff
@@ -10,6 +10,8 @@
   "claims": [
     "f7d7d4e4-255e-46dc-9f08-b953fb1e872d"
   ],
+  "checkin_items": false,
```
Comment (craigmcnally) [inline src/test/resources/po_line.sample]: again, https://github.com/folio-org/mod-orders-storage/pull/90 renames these files and refactors them to use camelCase.  You'll have to adjust once those changes are merged

Comment (craigmcnally) [reply src/test/resources/po_line.sample]: I don't see `"checkinItems": false` in the (renamed) `po-line.sample`

Comment (pautau-s) [reply src/test/resources/po_line.sample]: Missed. Thanks

**Pull request merged**